functions/exercise5/script.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+function convertWordFirstLetterIntoUpperCase(string){
+    var array = string.split(" ");
+    for(var i = 0; i < array.length; i++){
+        array[i] = array[i][0].toUpperCase() + array[i].substring(1);
```

Comment:
Accessing array[i][0] is quite risky.
You may have array[i] emty. In that case you should do nothing.